test/server/worker/workermanager.spec.js

```diff
@@ -36,16 +36,11 @@ describe('ServerWorkerManager - SimpleWorkers', function () {
             projectId: projectId,
             commitHash: ir.commitHash
         }, function (err, result) {
-            try {
-                expect(err).to.equal(null);
```

Comment:
For whatever reason, in node 18 the wrapped error message within the Assertion error got truncated. So the error message checks in the tests didn't match anymore